src/main/java/leduc/Parser.java

```diff
@@ -124,8 +124,13 @@ else if(user.matches(HelpCommand.getHelpShortcut())){
         else if (user.matches(ByeCommand.getByeShortcut())){
             c = new ByeCommand(user);
         }
+  
+        else if (user.matches(("stats"))){
```

Comment:
shortcut did not taken into account like other comments

Reply:
shortcut did not taken into account like other command* 
(Some methods should be add to Statscommand also